features/com.wso2telco.dep.hub.core.feature/src/main/resources/config/user-masking.properties

```diff
@@ -0,0 +1,16 @@
+# This Property is to enable or disable msisdn masking feature. set 'true' to enable feature
+user.masking.feature.enable=false
+
+# This Property MUST be set if UserAnonymization is set to 'true'
+# Production or sandbox esb payment url
+user.masking.feature.esb.payment.endpoint.url={http://esb-host}:{esb-nio-port}/payment
```

Comment:
Remove this property and generate URL from carbon util. Please check with ADS implementation